openshift_images/using_images/using-s21-images.adoc

```diff
@@ -9,14 +9,22 @@ You can use the link:https://access.redhat.com/documentation/en-us/red_hat_softw
 
 S2I images include:
 
+* .NET
 * Java
+* Go
 * Node.js
 * Perl
 * PHP
 * Python
 * Ruby
 
-S2I images are available for you to use directly from the {product-title} web UI by selecting *Catalog* -> *Developer Catalog*.
+S2I images are available for you to use directly from the {product-title} web console by following procedure:
+
+. Log in to the OpenShift Container Platform web console using your login credentials. The default view for the OpenShift Container Platform web console is the *Administrator* perspective.
```

Comment:
Requested edit, for every instance of OpenShift Contain Platform:
s/OpenShift Contain Platform/{product-title} 

If you prefer, I can open a follow-up PR to make this change.

Reply:
Updated, thanks for the feedback.